.github/workflows/go.yml

```diff
@@ -218,12 +218,14 @@ jobs:
         shard_total: [5]
         include:
           - { task: "mongodb", shard_index: 1, shard_total: 1 }
+          - { task: "yugabytedb", skip_flaky_check: true } # Too many failures for now
 
     uses: ./.github/workflows/_integration.yml
     with:
       task: ${{ matrix.task }}
       shard_index: ${{ matrix.shard_index }}
       shard_total: ${{ matrix.shard_total }}
+      skip_flaky_check: ${{ matrix.skip_flaky_check || false }}
```

Comment:
@noisersup why do you need to set skip_flaky_check for yugabytedb? Why does it need special handling?

Reply:
Ok, that's because UI on flaky test check becomes unuable.